components/content-service-api/initializer.proto

```diff
@@ -120,3 +121,15 @@ message GitStatus {
     // the total number of unpushed changes
     int64 total_unpushed_commits = 8;
 }
+
+// An additional content initializer allows to put files on top of anything that has been initialized before.
+// This is applied on top of any other initializers
+message AdditionalContentInitializer {
+    // files_url is a URL where one can download the file contents encoded as a JSON object where the relative pathes are the properties and contents are its values
```

Comment:
After reading through the code I understand the intention of making this a "JSON" field. I don't think this is the right way going forward because the JSON format is reasonably undefined. Until now the API is completely well defined through protobuf. Going this route does away with that and defers validation and structure. We could instead define the file list in protobuf (see comment above).

Re OTS integration: what I had in mind is that we serve the individual file content using OTS and provide them to a `FileDownloadInitializer` as outlined above - rather than to provide a single file that gets downloaded.